readme.txt

```diff
@@ -120,6 +120,8 @@ Currently, the following add-ons are available for Event Tickets:
 = [4.10.2] TBD =
 
 * Tweak - Use buttons instead of links and add better feedback on checkin (disable buttons) [70618]
+* Tweak - Use get_stylesheet_directory() instead of get_template_directory() to honor child themes for AR template [123613]
+* Fix - array_values() should be array_keys() as get_templates() is keyed by filename, not template name. Props to @eri-trabiccolo for flagging this! [123613]
```

Comment:
```suggestion
* Fix - Resolve problems with WP_Theme::get_page_templates() usage, use array_keys() instead of array_values() since the array is keyed by filename, not template name. Props to @eri-trabiccolo for flagging this! [123613]
```